2.4/Dockerfile

```diff
@@ -37,7 +34,6 @@ RUN yum install -y epel-release && \
     yum install -y --setopt=tsflags=nodocs bind-utils gettext iproute v8314 mongodb24-mongodb mongodb24 && \
     yum clean all && \
     mkdir -p /var/lib/mongodb/data && chown -R mongodb.mongodb /var/lib/mongodb/ && \
```

Comment:
@mnagy line 37 below covers write permission to /var/lib/mongodb for everyone else which is why this works for any user.